src/invocation/ConnectionAuthenticator.ts

```diff
@@ -42,16 +50,27 @@ export class ConnectionAuthenticator {
             .invokeOnConnection(this.connection, credentials)
             .then((msg: ClientMessage) => {
                 var authResponse = ClientAuthenticationCodec.decodeResponse(msg);
-                if (authResponse.status === 0) {
-                    this.connection.setAddress(authResponse.address);
-                    this.connection.setConnectedServerVersion(authResponse.serverHazelcastVersion);
-                    if (ownerConnection) {
-                        this.clusterService.uuid = authResponse.uuid;
-                        this.clusterService.ownerUuid = authResponse.ownerUuid;
-
-                    }
-                } else {
-                    throw new Error('Could not authenticate connection to ' + this.connection.getAddress().toString());
+                switch (authResponse.status) {
+                    case AuthenticationStatus.AUTHENTICATED:
+                        this.connection.setAddress(authResponse.address);
+                        this.connection.setConnectedServerVersion(authResponse.serverHazelcastVersion);
+                        if (ownerConnection) {
+                            this.clusterService.uuid = authResponse.uuid;
+                            this.clusterService.ownerUuid = authResponse.ownerUuid;
+
+                        }
+                        this.logger.debug('ConnectionAuthenticator',
+                            'Connection to ' +
+                            this.connection.getAddress().toString() + ' authenticated');
+                        break;
+                    case AuthenticationStatus.CREDENTIALS_FAILED:
+                        this.logger.debug('ConnectionAuthenticator', 'Invalid Credentials' );
```

Comment:
`debug` is very low level logging. default logger does not even show debug output. this information should be at `error` level.